src/applications/claims-status/containers/FilesPage.jsx

```diff
@@ -102,9 +102,9 @@ class FilesPage extends React.Component {
     );
 
     documentsTurnedIn.push(...supportingDocuments);
-    documentsTurnedIn.sort((dateA, dateB) => {
-      if (dateB === dateA) return 0;
-      return dateB > dateA ? -1 : 1;
+    documentsTurnedIn.sort((a, b) => {
+      if (a.date === b.date) return -1;
```

Comment:
Why return -1 if they're equal?

Reply:
The way the EVSS serializer works is by sorting the whole array from least to most recent, then reversing the array (most to least recent). By sorting from most to least recent I am cutting out the extra step, with the caveat that items that have the same date end up staying in the same order, where EVSS would end up reversing those items. I found that by using -1, they tend to display in the same order as with EVSS

Reply:
Interesting, thanks for clarifying. This is sort of unintuitive behavior for a custom sorter - do we think it's gonna be a big burden for Veterans if we keep equality as it was to avoid developer confusion in the future?

Reply:
My main goal at the moment is to preserve the EVSS behavior (for better or worse). In order to ensure that the migration goes smoothly, I think it would be ideal if when switching between the EVSS and Lighthouse version of the API, the UI is virtually identical. That being said, I think that the design efforts that are going on now will probably necessitate a rewrite of this whole component anyways, so I'm not super concerned about the long term effects, especially since this code is less than a day old anyways

Reply:
Works for me.